Description: use the created java header/libs dir and don't clean it
Author: Johann Felix Soden <johfel@gmx.de>

--- a/pdftk/Makefile.Base
+++ b/pdftk/Makefile.Base
@@ -14,7 +14,7 @@
 # platform-specific makefiles, e.g.: Makefile.Debian
 #
 
-export JAVALIBPATH= ../java_libs_lgpl
+export JAVALIBPATH= ../java_libs2
 
 JAVALIB= $(JAVALIBPATH)/java_lib.o
 
@@ -29,7 +29,7 @@
 #
 CPPFLAGS+= -DPDFTK_VER=\"1.43\"
 
-all : javalib pdftk
+all : pdftk
 
 javalib :
 	$(MAKE) -f Makefile -iC $(JAVALIBPATH) all
@@ -55,4 +55,3 @@
 clean:
 	$(RM) $(RMFLAGS) *.o
 	$(RM) $(RMFLAGS) pdftk
-	$(MAKE) -f Makefile -C $(JAVALIBPATH) clean
